package.json

```diff
@@ -1,57 +1,25 @@
 {
-  "name": "bifold-main",
-  "version": "0.0.1",
-  "description": "Aries Mobile Agent React Native is an open source Aries mobile agent app. This is a project that has been created to focus the community's efforts towards a central open source project. Various different organizations and people have expressed interest in a open source community project to help focus efforts to help prevent duplication of work between projects. Aries Bifold is also intended to help get complex or specific use-case projects started faster by cloning and having a basis of an Aries agent.",
-  "main": "entry.js",
+  "name": "aries-bifold",
   "private": true,
-  "directories": {
-    "doc": "docs",
-    "lib": "lib"
-  },
-  "workspaces": {
-    "packages": [
-      "core",
-      "app"
-    ]
-  },
-  "engines": {
-    "node": ">=16.15.0 <18.0.0",
-    "yarn": "^3.3.1"
-  },
-  "resolutions": {
-    "@types/react": "17.0.2",
-    "@types/react-dom": "17.0.2",
-    "@react-navigation/core": "6.1.0",
-    "@react-navigation/native": "6.0.6",
-    "@react-navigation/stack": "6.0.11",
-    "@types/node": "16.9.6",
-    "@aries-framework/core": "0.3.3"
+  "license": "Apache-2.0",
+  "workspaces": [
+    "packages/oca",
+    "packages/core",
+    "packages/legacy"
+  ],
+  "repository": {
+    "url": "https://github.com/hyperledger/aries-mobile-agent-react-native",
+    "type": "git"
   },
   "scripts": {
-    "ci:core": "cd core && npm ci",
-    "ci:app": "cd app && npm ci",
-    "ci:all": "",
-    "test": "cd core && npm test",
-    "build:core": "yarn workspace aries-bifold run prepack",
-    "build:app": "",
-    "build:all": "run-s build:core build:app",
-    "prepare": "run-s ci:all build:all",
-    "lint:core": "cd core && npm run lint --if-present",
-    "lint": "run-s lint:core",
-    "prettier:core": "cd core && npm run prettier --if-present",
-    "prettier": "run-s prettier:core",
-    "start": "cd app && npm run start",
-    "android": "cd app && npm run android",
-    "ios": "cd app && npm run ios",
-    "pre-commit-lint:core": "cd core && npm run pre-commit-lint",
-    "pre-commit-lint": "run-s pre-commit-lint:core",
-    "typecheck:core": "cd core && npm run typecheck",
-    "typecheck": "run-s typecheck:core"
+    "prettier": "run-s prettier",
+    "test": "cd packages/legacy/core && npm test"
   },
-  "author": "",
-  "license": "Apache-2.0",
   "devDependencies": {
-    "npm-run-all": "^4.1.5"
+    "prettier": "^2.8.4"
   },
-  "packageManager": "yarn@3.3.1"
```

Comment:
can we keep `packageManager` entry for now? This is managed by corepack when enabling/configuring packaging manager.